### ISSUE TITLE
Barrierefreie Erstellung von Inhalten - Alternativtext bei eingefügten Bildern

### ISSUE THREAD

**Issue by Dottopia** [BIK-BITV/BIK-Web-Test#445]
Hallo,
mich beschäftigt derzeit eine Frage zum setzen eines Alternativtextes.
Folgendes Szenario: 
Beim hochladen eines Bildes auf eine Webanwendung, wird ein Modal angezeigt. In diesem Modal kann man ein Alternativtext schreiben oder eine Checkbox setzen, welche das Bild als "dekorativ" markiert und dementsprechend ein leeres Alt-Attribut setzt. Ähnlich wie bei Word.

Jetzt kam die Frage auf, ob das dekorativ markieren überhaupt erlaubt ist, nach dem Prüfschritt? Denn wenn eine Person ein Bild hochlädt, welches eine Information bereitstellt und dieses dekorativ markiert, dann ist das ja nicht barrierefrei. Anders sieht es doch aus, wenn dieses Bild zusätzlich im Text beschrieben ist und das Bild nur Zusatz ist.

Dankeschön

**Comment by MarcHaunschild** [comment]
Und wenn es nur Deko ist? und wenn es im Fließtext beschrieben werden soll?

Dann macht es doch Sinn, es als dekorativ zu kennzeichnen?!?

**Comment by Dottopia** [comment]
Dann ja. Allerdings kann ich als Betreiber der Webanwendung doch nicht garantieren, ob die Autoren, welche das Bild bzw. den Screenshot hochladen, es im Fließtext ausreichend beschreiben oder das Bild wirklich nur Deko ist?

**Comment by MarcHaunschild** [comment]
Nein. Genau darum **muss** es ja die Person entscheiden, die das Bild hochlädt. MS weiß ja auch nicht, ob du ein dekoratives Bild in Word einsetzt und bietet daher die Checkbox, damit man eine Möglichkeit hat, dekorative Bilder zu kennzeichnen.

Gerade **weil** die Software es nicht wissen kann, benötigt man als Mensch diese Option!

**Comment by Dottopia** [comment]
Das ist richtig, daher habe ich diese Option. 
Aber ich habe ja gar kein Einfluss darauf, ob eine Person nun das hochgeladene Bild im Fließtext ausreichend beschreibt und oder, ob Sie, zb. aus Faulheit, das Bild immer als dekorativ setzt. 
Ist dadurch mein Produkt dann nicht barrierefrei?

**Comment by sweckenmann** [comment]
Hängt es nicht auch vom Kontext ab? Ist es eine Fachanwendung, wo bestimmte Bilder hochgeladen werden sollen (etwa abfotografierte Dokumente), die in keinem Kontext haben, dann ist ein Alternativtext zwingend notwendig?

Ansonsten kann man bei Autorewerkzeugen nur bestmögliche Voraussetzungen schaffen, um es Autoren zu ermöglichen barrierefreie Inhalte zu erstellen.

**Comment by pkra** [comment]
Mir erscheint diese Art der UI immer wie eine Art Kategorienfehler. Alternativtexte sind kontextabhängig und beim Hochladen in eine Art Mediensammlung fehlt fast immer der Kontext )bzw. solche Sammlungen existieren ja meist, um Verwendung in mehreren Kontexten zu erleichtern). 

Beim Einfügen von Bildern scheint mir diese Art UI wesentlich sinnvoller (wie z.B. auf sozialen Medien). Dann ist der Kontext klar(er) und ermöglicht auch unterschiedliche Alternativtexte für dasselbe Bild.

**Comment by MarcHaunschild** [comment]
> Aber ich habe ja gar kein Einfluss darauf, ob eine Person nun das hochgeladene Bild im Fließtext ausreichend beschreibt und oder, ob Sie, zb. aus Faulheit, das Bild immer als dekorativ setzt. 
> 
> Ist dadurch mein Produkt dann nicht barrierefrei?

Mit Produkt meinst du offenbar ein CMS oder so ähnlich.

Das ist (wie das hier bereits erwähnte Word) immer nur ein Werkzeug um etwas barrierefreies zu erstellen.

Dafür muss dein Produkt bestimmte Möglichkeiten bieten. Zum Beispiel eben die einem Bild einen alternativ-Text mitzugeben oder es als dekorativ auszuzeichnen und diese Angaben dann sinnvoll auszugeben.

Das was mit deinem Werkzeug erstellt wird, ist auch wieder ein eigenständiges Produkt. So wie jedes Word- oder PDF-Dokument und kann auf Barrierefreiheit geprüft werden.

Ob dein Produkt barrierefrei ist, kann man auch prüfen. Dazu muss dann dein CMS selber von Menschen mit Behinderungen benutzt werden können.

Also zwei unterschiedliche Dinge: dein Produkt und das Produkt, was unter Zuhilfenahme deines Werkzeuges erstellt wird.

**Comment by Dottopia** [comment]
Mein Produkt ist nicht ganz ein CMS, aber es beinhaltet Rich-Text-Editoren, welche es ermöglichen unter Anderem Bilder in den Editor hochzuladen. Diese Bilder sind immer Screenshots von bestimmten Anwendungsfälle. Natürlich sind die Nutzenden dazu angehalten, das Problem textuell zu beschreiben und das Bild nur zusätzlich ist. Aber das kann ja nie garantiert werden.

Sehe ich es richtig, dass durch die Möglichkeit ein hochgeladenes Bild dekorativ zu markieren oder diesem einen ausführlichen Alternativtext zu geben, mein Produkt/Webanwendung hinsichtlich Autorenwerkzeug barrierefrei ist? Und wenn die Autoren dies falsch verwenden, dies keine Auswirkung auf meine BITV-Konformität hat?

**Comment by MarcHaunschild** [comment]
In Bezug auf Konformität reicht das auf jeden Fall.
Aber schöner wäre es sicher, wenn man noch eine Bildunterschrift vergeben könnte...

**Comment by Dottopia** [comment]
Zusätzlich zu einem Alternativtext eine Bildunterschrift? Was genau sollte dann in den Alternativtext und was in die Bildunterschrift?

**Comment by MarcHaunschild** [comment]
Na ja, es gibt ja keine Pflicht die text-Alternative ins Alt-Attribut zu schreiben.
Insbesondere wenn längere Beschreibungen nötig werden sollten die möglichst nicht da rein.
Aber das führt hier zu weit und wäre Teil z.B. einer Redaktions-Schulung. Nur so viel: am Ende muss ein in sich schlüssiger Text aus dem Screenreader kommen, der auch in der Lese-Reihenfolge Sinn ergibt und möglichst ohne Redundanz auskommt.

**Comment by johannesFischer84** [comment]
> Sehe ich es richtig, dass durch die Möglichkeit ein hochgeladenes Bild dekorativ zu markieren oder diesem einen ausführlichen Alternativtext zu geben, mein Produkt/Webanwendung hinsichtlich Autorenwerkzeug barrierefrei ist? Und wenn die Autoren dies falsch verwenden, dies keine Auswirkung auf meine BITV-Konformität hat?

ja, das Autorenwerkzeug ist in dieser Hinsicht dann barrierefrei. Bzw. ist der Webauftritt bezüglich seiner Autorenwerkzeuge und deren Anforderungen (EN 11.8.x) für Bilder nach EN 301 549 barrierefrei. Die Autoren müssen aber die Funktion korrekt verwenden, damit der Webauftritt bezüglich von Alternativtexten von Abbildungen barrierefrei ist/bleibt. Die Autoren müssen also geschult sein. Das Autorenwerkzeug sollte die dekorative Option aber beinhalten, sonfern dekorative Inhalte nicht gänzlich ausgeschlossen werden können.

> Zusätzlich zu einem Alternativtext eine Bildunterschrift?

Eine Bildunterschrift ist für die Barrierefreiheit nicht erforderlich. Sofern eine Bildunterschrift vorhanden ist, sind dort meist zum Bild zusätzliche Informationen enthalten, also Informationen, die sehende Menschen allein aus dem Bild nicht unbedingt entnehmen können. Ein Alternativtext enthält dagegen meist die Information, die sehenden Menschen allein aus dem Bild erkennen können.

**Comment by MarcHaunschild** [comment]
> Eine Bildunterschrift ist für die Barrierefreiheit nicht erforderlich. Sofern eine Bildunterschrift vorhanden ist, sind dort meist zum Bild zusätzliche Informationen enthalten, also Informationen, die sehende Menschen allein aus dem Bild nicht unbedingt entnehmen können.

Du hast recht: sichtbare Alternativen für Bilder sind für die Erfüllung formaler Kriterien der Barrierefreiheit nicht notwendig!

Aber erlaube mir eine Ergänzung, um ein wenig zu sensibilisieren:

Ich kann mir kaum ein Bild vorstellen auf das es zutrifft, dass alle Sehenden jede relevante Information aus einem Bild **unbedingt** und auch noch in gleicher Weise entnehmen (können).

Ablenkung, also mangelnde Konzentration, unterschiedliche Assoziationen aufgrund eines anderen kulturellen Backgrounds oder auch kognitive Einschränkungen können zu deutlich unterschiedlichen Wahrnehmungen führen. Oder Stress: Wer mal einer Zeugenvernehmung zugehört hat, weiß was ich meine. Da können sich von zehn Leuten keine zwei auf einen Unfallhergang einigen...

Selbst eine Lupe bedeutet für jemanden, der schlecht sieht und ständig zoomt, womöglich etwas anderes als für eine andere die ständig mit der Suchfunktion in Produktlisten arbeitet...

Sichtbare Textalternativen sind im barrierefreien Umfeld immer ein Gewinn, nicht nur für die leider in den Normen sehr schwach repräsentierten Menschen mit kognitiven Einschränkungen oder neurodiverse Personen...

**Comment by sweckenmann** [comment]
Ist hier nicht das Thema aber ich sehe es auch wie @johannesFischer84. Alternativtexte und Bildunterschriften haben meiner Ansicht nach unterschiedliche Aufgaben. Ein Alternativtext beschreibt das Bild und eine Bildunterschrift setzt das Bild in einen Kontext. Ich würde @MarcHaunschild bei Symbolschaltflächen zustimmen, hier würde eine sichtbare Beschriftung einen Mehrwert bedeuten (wenn nicht sicherzustellen ist, dass alle das Symbol kennen), aber dann wäre die Grafik ja als dekorativ auszuzeichnen. Sonst läuft man Gefahr Redundanzen zu erzeugen (wenn alt und Bildunterschrift gleich sind), was nicht dramatisch ist, aber wir sprechen ja über Best Practice.

Ein Beispiel aus einem Praxis-Leitfaden vom DVBS: 
- ALT: Ein Rudel mit fünf Wölfen in einem kahlen Wald
- BIldunterschrift: In Fabeln symbolisiert Isegrim Kraft, Rücksichtslosigkeit, Gier und Bösartigkeit.

**Comment by detlevhfischer** [comment]
@Dottopia Ich denke aus dem Issue ergibt sich kein Handlungsbedarf bez. Prüfschritt und schließe es deshalb. Ich hoffe, die längere Diskussion war hilfreich.
